src/lib/es5.d.ts

```diff
@@ -430,7 +430,7 @@ interface String {
     /**
      * Replaces text in a string, using a regular expression or search string.
      * @param searchValue A string to search for.
-     * @param replaceValue A string containing the text to replace for every successful match of searchValue in this string.
+     * @param replaceValue A string containing the text to replace. When the searchvalue is a string, only the first match is replaced. If the searchValue is a Regexp, all matches are replaced if the g flag is set. Otherwise only the first one is.

```

Comment:
I think there might be syntax for referring to another parameter (i.e. in a way that will work with find-all-refs or rename) - @sandersn  would know.

Reply:
The syntax is `{@link searchValue}`, although maybe `{@linkcode searchValue}` would be better since it usually produces the `monospace` font used for code.